### ISSUE TITLE
Implement scroll wheel for zooming in new validation UI?

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#3594]
Could save clicks and be more efficient/seamless. Also matches how GSV works.

![image](https://github.com/ProjectSidewalk/SidewalkWebpage/assets/1621749/5f2d2a07-b9ca-4ac2-91b2-e10d8743bcc1)

**Comment by misaugstad** [comment]
I'm guessing that it wouldn't be too hard to do what we're doing now with the scroll wheel.

If we wanted to have a more continuous zoom with the scroll wheel, I'd need to do some digging and a lot of testing to make sure that our whole pipeline would work with the different zoom levels. The integer zoom levels may have been set on purpose due to how we download panos/crops from Google.

**Comment by misaugstad** [comment]
We talked about this briefly at a 1:1 and consider it to be low priority.

**Comment by jonfroehlich** [comment]
Just came here to comment that I miss not having this; I naturally try to use my mouse scroll wheel for Zoom because that's a native interaction technique in GSV...

**Comment by misaugstad** [comment]
We could likely start with adding in non-integer zoom everywhere but the Explore page. For the Explore page, we'd need to do a lot more testing to make sure the whole pipeline works with making crops and such.

**Comment by jonfroehlich** [comment]
Right, I was specifically talking about Validation. But could also add to Gallery, etc.

Understood re: Explore. But maybe with https://github.com/ProjectSidewalk/SidewalkWebpage/issues/3839, we could actually do continuous Zoom in Explore too?

**Comment by misaugstad** [comment]
>Understood re: Explore. But maybe with https://github.com/ProjectSidewalk/SidewalkWebpage/issues/3839, we could actually do continuous Zoom in Explore too?

Yeah, that would make this a lot easier! Would still need to be more careful with testing, but it removes a big obstacle
